src/pages/billing/Project.js

```diff
@@ -564,29 +564,36 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
 
   const tabToTable = {
     workspaces: h(Fragment, [
-      div({ role: 'table', 'aria-label': `workspaces in billing project ${billingProject.projectName}` }, [
-        h(WorkspaceCardHeaders, {
-          needsStatusColumn: billingAccountsOutOfDate,
-          sort: workspaceSort,
-          onSort: setWorkspaceSort
-        }),
-        div({}, [
-          _.flow(
-            _.orderBy([workspaceSort.field], [workspaceSort.direction]),
-            _.map(workspace => {
-              const isExpanded = expandedWorkspaceName === workspace.name
-              return h(WorkspaceCard, {
-                workspace: { ...workspace, billingAccountDisplayName: billingAccounts[workspace.billingAccount]?.displayName },
-                billingProject,
-                billingAccountStatus: billingAccountsOutOfDate && getBillingAccountStatus(workspace),
-                key: workspace.workspaceId,
-                isExpanded,
-                onExpand: () => setExpandedWorkspaceName(isExpanded ? undefined : workspace.name)
+      !_.isUndefined(workspaces) && _.isEmpty(workspacesInProject) ?
+        div({ style: { ...Style.cardList.longCardShadowless, width: 'fit-content' } },
+          [span({ 'aria-hidden': 'true' }, ['Use this Terra billing project to create']),
+            h(Link, {
```

Comment:
Changed to an actual link instead of button (which means you need to supply an href), and accessible text because otherwise all you get is "Workspaces", unconnected from the text that comes before it.

Reply:
I see, I was confused about that but it makes a lot more sense now! Thanks for making the changes